<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorWindow</class>
 <widget class="QMainWindow" name="GeneratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Генератор схем</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="elementsNumberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Число элементов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="elementsNumberText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="nodeCapacityGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Разрядность узлов</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="nodeCapacityMeanLabel">
       <property name="text">
        <string>Медиана</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nodeCapacityMeanText">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nodeCapacitySigmaLabel">
       <property name="text">
        <string>Отклонение</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nodeCapacitySigmaText">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nodeCapacityLeftLimitLabel">
       <property name="text">
        <string>Левая граница</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nodeCapacityLeftLimitText">
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="nodeCapacityDescLabel">
       <property name="text">
        <string>Значения указываются в степенях двойки</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="nodeCapacityRightLimitLabel">
       <property name="text">
        <string>Правая граница</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nodeCapacityRightLimitText">
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="branchingGroup">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Разветвленность</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="branchingMeanLabel">
       <property name="text">
        <string>Медиана</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="branchingMeanText">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="branchingSigmaLabel">
       <property name="text">
        <string>Отклонение</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="branchingSigmaText">
       <property name="text">
        <string>0.5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="branchingLeftLimitLabel">
       <property name="text">
        <string>Левая граница</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="branchingRightLimitLabel">
       <property name="text">
        <string>Правая граница</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="branchingLeftLimitText">
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="branchingRightLimitText">
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="librariesLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Библиотеки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="librariesText">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="librariesButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерировать</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
